ui/test/nuxeo-actions-menu.test.js

```diff
@@ -123,54 +163,23 @@ suite('<nuxeo-actions-menu>', () => {
     expect(dropdownButton(menu).hidden).to.be.true;
   });
 
-  test('menu with slotted content', async () => {
-    const menu = await fixture(html`
-      <div style="max-width: 160px;">
-        <nuxeo-actions-menu>
-          <nuxeo-slot slot="SLOT"></nuxeo-slot>
-        </nuxeo-actions-menu>
-      </div>
-    `);
-    await flush();
-
-    await fixture(html`
-      <nuxeo-slot-content name="content" slot="SLOT">
-        <template>
-          ${[...Array(5)].map(
-            () => html`
-              <nuxeo-preview-button document='{ "entity-type": "document", "facets": [] }'></nuxeo-preview-button>
-            `,
-          )}
-        </template>
-      </nuxeo-slot-content>
-    `);
-
-    await flush();
+  test('menu with nuxeo-slotted content', async () => {
+    const menu = await makeMenuWithNuxeoSlot();
+    await makeNuxeoSlottedMenuContent();
 
     expect(menuActions(menu).length).to.be.equal(3);
     expect(dropdownActions(menu).length).to.be.equal(2);
     expect(dropdownButton(menu).hidden).to.be.false;
   });
 
-  test('menu with unresolved slotted content', async () => {
-    const menu = await fixture(html`
-      <div style="max-width: 160px;">
-        <nuxeo-actions-menu>
-          <nuxeo-slot slot="SLOT"></nuxeo-slot>
-        </nuxeo-actions-menu>
-      </div>
-    `);
-    await flush();
+  test('menu with unresolved nuxeo-slotted content', async () => {
+    const menu = await makeMenuWithNuxeoSlot();
 
     await fixture(html`
       <nuxeo-slot-content name="content" slot="SLOT">
         <template>
           <nuxeo-test-button icon="icons:home"></nuxeo-test-button>
```

Comment:
What was the purpose of having a new custom element here?

Reply:
Note that this was just a cleanup to a test that was already here. The goal of this test with an element that will only be defined *after* it was contributed to the slot, i.e., that will be `unresolved` when it's contributed.